src/main/java/lotto/domain/prize/Prize.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain.prize;
+
+import java.util.Arrays;
+
+public enum Prize {
+
+    MISS(0, 0),
+    FIFTH(3, 5000),
+    FOURTH(4, 50000),
+    THIRD(5,  1500000),
+    FIRST(6,  2000000000);
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000);
```
java7부터는 위처럼 선언해서 가독성을 높일수 있다고 해요 ㅎㅎ